golden_data.Rmd

```diff
@@ -1,25 +1,51 @@
+
 ## Alternative sources
 
-EggNOG and FunGene are my favourite sources, though this leaves out the vast majority of biological sequence databases for no reason other than simplicity. IMG, KEGG, UniProt, Pfam, and other similar databases are all great but they lack an API to retrieve lineage information, and therefore this needs to be provided in either a separate table or the FASTA headers.
+EggNOG and FunGene are my favourite sources, though this leaves out the majority of biological sequence databases for no reason other than simplicity. IMG, KEGG, Pfam, and other similar databases are all great but they lack an API to retrieve lineage information, and therefore this needs to be provided in either a separate table.
 
-First, the table. The lineage table can be provided using the `--seqs2lineage` argument. The format of the lineage table is tabular or comma-separated with the fields:
+A table that maps each sequence identifier to its respective taxonomic lineage can be provided using the `--seqs2lineage` argument. The format of the lineage table is tabular or comma-separated with the fields:
 
 | SeqID | Organism | Lineage | Domain | Phylum | Class | Order | Family | Genus | Species |
 |-------|----------|---------|--------|--------|-------|-------|--------|-------|---------|
 
 'SeqID' must be the first field and it can be a prefix of the query sequences that were placed. This is useful when the queries are contigs (i.e. nucleotide sequences) but the amino acid open-reading frames were classified. Providing the contig name as the 'SeqID' will work. Some of these fields are redundant and not all are required. If the "Lineage" field is provided, the separated rank fields (e.g. "Domain", "Phylum", etc.) are not needed and vice versa. If the lineage field is used there are two things that can help you and TreeSAPP: first off, other non-canonical ranks (e.g. sub-order, super-phylum, etc.) should be removed from the lineage. Secondly, it is best to explicitly include the rank-prefixes in the lineages (d\_\_Archaea; p\_\_Euryarchaeota) so TreeSAPP doesn't have to do this later. The "Organism" field is also optional and the most resolved rank is used in its place if it is missing.
 
-The second option, involves modifying the sequence's FASTA header to follow the format: \>SeqID lineage=cellular organisms; Domain; Phylum; Class [Organism_name]
+## Data access hazards
+
+Accessing the fraction of well-curated data from multiple databases is not trivial. Perhaps the most obvious reason for this are unrelated genes with shared names. For example, the gene product of mcrB in _E. coli_ widely refers to 5-methylcytosine-specific restriction enzyme B, but in the Archaea it would refer to the beta subunit of methyl-coenzyme M reductase - a completely unrelated protein. This is an inevitability in a field of science as broad as biology, where scientists that are responsible for naming genes cannot possibly be aware of all gene names in circulation.
```

Comment:
```suggestion
Accessing the fraction of well-curated data from multiple databases is not trivial. Perhaps the most obvious reason for this are unrelated genes with shared names. For example, the gene product of mcrB in _E. coli_ widely refers to 5-methylcytosine-specific restriction enzyme B, but in the Archaea, it would refer to the beta subunit of methyl-coenzyme M reductase - a completely unrelated protein. This is an inevitability in a field of science as broad as biology, where scientists that are responsible for naming genes cannot possibly be aware of all gene names in circulation.
```